src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/DefaultEditorTemplates.cs

```diff
@@ -245,6 +248,7 @@ public static IHtmlContent ObjectTemplate(IHtmlHelper htmlHelper)
             }
 
             var serviceProvider = htmlHelper.ViewContext.HttpContext.RequestServices;
+            var htmlEncoder = serviceProvider.GetRequiredService<HtmlEncoder>();
```

Comment:
@Eilon this is a fairly ugly option. Other possibilities:

1. Using Razor's `NullHtmlEncoder` would avoid an additional service locator and avoid useless HTML encoding but it would also introduce a new Microsoft.AspNetCore.Mvc.ViewFeatures.dll dependency. I.e. that approach may break MVC Core apps that `AddViews()` but do not `AddRazorViewEngine()`.
2. Could instead move the generally-useful `NullHtmlEncoder` into the HtmlAbstractions repo and its Microsoft.Extensions.WebEncoders project. That would be similar to `HtmlTestEncoder` but wouldn't use the `Microsoft.Extensions.WebEncoders.Testing` namespace. Unfortunately it would cause ambiguities in any code that used `NullHtmlEncoder` and referenced both of the Microsoft.Extensions.WebEncoders and Microsoft.AspNetCore.Razor.Runtime packages.
3. Could duplicate the 120 lines of `NullHtmlEncoder` as a `private` class in this file. Also icky due to the code duplication.

What is your favourite choice? Any other ideas?